src/components/components/ebay-tooltip-base/component-browser.ts

```diff
@@ -40,7 +40,9 @@ class TooltipBase extends Marko.Component<Input> {
 
     handleExpand() {
         this.emit("base-expand");
-        this.updateTip();
+        if (this.hostEl && this.overlayEl) {
```

Comment:
Would it make sense to move this check into the `updateTip` function instead of adding it as a wrapper each time `updateTip` is called?

Reply:
We can refactor, but the reason is because it already is being checked on setupExpander
```
        if (this.hostEl && this.overlayEl) {
            this.updateTip();
            this.cleanup = autoUpdate(
                this.hostEl,
                this.overlayEl,
                this.update.bind(this),
            );
        }
```
And we dont want to call autoUpdate without hostEl and overlayEl being defined. 
So I don't want this check to be called multiple times.